tests/suites/sanity/sanity_test.go

```diff
@@ -65,53 +83,23 @@ var _ = Describe(TestName, func() {
 
 		err = Operator.Instance.WaitForPlanComplete("deploy")
 		Expect(err).To(BeNil())
-
 		assertNumberOfCassandraNodes(NodeCount)
-	})
 
-	It("Upgrades the running operator instance from a directory", func() {
-		before, _, err := cassandra.OverrideOperatorVersion(TestOperatorVersion)
```

Comment:
you can remove the function `OverrideOperatorVersion` as after this removal it shouldn't be used anywhere else.

Reply:
Good point. I want to keep it for now as the upgrade test is coming back, as an extra test though